src/components/CoverTypeQuestion/CoverTypeQuestion.tsx

```diff
@@ -1,35 +1,34 @@
-import React, { useContext } from 'react';
+import React from 'react';
+import { Nullable } from '@constructor-io/constructorio-client-javascript/lib/types';
 import QuestionTitle from '../QuestionTitle/QuestionTitle';
-import QuizContext from '../CioQuiz/context';
 import QuestionDescription from '../QuestionDescription/QuestionDescription';
 import { renderImages } from '../../utils';
+import { QuestionImages } from '../../types';
 
-export default function CoverTypeQuestion() {
-  const { state } = useContext(QuizContext);
-  let question;
+export interface ICoverTypeQuestionProps {
```

Comment:
How would this work without the context and why are we only doing it for Cover?

Reply:
Just refactored one component so the PR is not big and also to make sure we agree on those changes before changing all of the components